### PR TITLE
make graph for durative action

### PR DESCRIPTION
**Pull request by h-kamada**
* fix durative action demo
* fix `make-graph-from-pddl-results` func for durative action

### CODE REVIEW

**Comment by h-kamada** [comment]
* align durative actions to timeline actions (one action is devided to start action and end action and they are aligned in time series.)
pddl_result

File: pddl/pddl_planner/demos/2016_kamada_durative/demo.l [hunk 1]
```diff
@@ -1,94 +1,61 @@
 #!/usr/bin/env roseus
-(require :pddl-result-graph "package://pddl_planner/src/pddl-result-graph.l")
-(require :eus-pddl-client "package://pddl_planner/src/eus-pddl-client.l")
+;; mostly same to demo-eus.l in pddl package under jsk subversion trunk
 
+(load "package://pddl_planner/src/pddl-result-graph.l")
+(load "package://pddl_planner/src/eus-pddl-client.l")
```
Comment (k-okada) [inline pddl/pddl_planner/demos/2016_kamada_durative/demo.l]: Why you chances from require to load

Comment (h-kamada) [inline pddl/pddl_planner/demos/2016_kamada_durative/demo.l]: this was unnecessary change. I fixed.

Comment (k-okada) [inline pddl/pddl_planner/demos/2016_kamada_durative/demo.l]: Ok, please let me know when Travis passed


On Saturday, 2 July 2016, h-kamada <notifications@github.com> wrote:

> In pddl/pddl_planner/demos/2016_kamada_durative/demo.l
> <https://github.com/jsk-ros-pkg/jsk_planning/pull/47#discussion_r69374479>
> :
>
> >
> > +(load "package://pddl_planner/src/pddl-result-graph.l")
> > +(load "package://pddl_planner/src/eus-pddl-client.l")
>
> this was unnecessary change. I fixed.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/jsk-ros-pkg/jsk_planning/pull/47/files/4f50dd668c6d254b0495a5b217c015b907374b6e#r69374479>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAeG3EpRTp05AomSzEPNlPTHoHr1JG1Uks5qRhZ9gaJpZM4JC2Jp>
> .
>


-- 
--
◉ Kei Okada

**Comment by h-kamada** [comment]
All checks have passed. Please merge.

**Comment by h-kamada** [comment]
All checks have passed. Please merge.

**Comment by h-kamada** [comment]
@k-okada , Please merge.
